### DISCUSSION TITLE
Functional Description of Channel Types

### DISCUSSION THREAD

**Discussion by ShahanaFarooqui** [ACINQ/eclair#2489] [Q&A] [answered]
I want to explain channel types in tooltips for our users, which can help them to better understand these types. Could you please give a short functional description of standard, static_remotekey, anchor_outputs_zero_fee_htlc_tx channel types?

**Comment by t-bast** [comment, marked as answer]
I'm not sure it's a good idea. We currently support various channel types, but it's only for backwards-compatibility purposes, `anchor_outputs_zero_fee_htlc_tx` is superior to all the other alternatives for node operators, so we should strongly nudge people towards using that one. I don't think a UI should even offer the choice to explicitly downgrade...

**Comment by ShahanaFarooqui** [comment]
@t-bast Thank you for the quick response. I removed the channel type selection. We will open all channels with type `anchor_outputs_zero_fee_htlc_tx` now.

**Reply by t-bast** [reply]
Actually it would be even better if you didn't explicitly specify a `channel type`. If you don't specify anything, we will automatically use the most modern one available based on our peer's features. If you do specify `anchor_outputs_zero_fee_htlc_tx` and the peer doesn't support it, the channel opening will just fail, which is bad UX. As `cln` still doesn't support `anchor_outputs_zero_fee_htlc_tx`, it's a bit early in my opinion to explicitly require it all the time.

**Reply by ShahanaFarooqui** [reply]
Thank you for the explanation. Removed channel type altogether for RTL.
